web/crux/src/filters/http-exception.filter.ts

```diff
@@ -0,0 +1,32 @@
+import { ExceptionFilter, Catch, ArgumentsHost } from '@nestjs/common'
+import { Request, Response } from 'express'
+
+interface Exception {
+  name: string
+  message: string
+}
+
+@Catch()
+export default class HttpExceptionFilter implements ExceptionFilter {
+  catch(exception: Exception, host: ArgumentsHost) {
+    const ctx = host.switchToHttp()
+    const response = ctx.getResponse<Response>()
+    const request = ctx.getRequest<Request>()
+
+    if (exception.name === 'NotFoundException') {
+      return response.status(404).json({
+        statusCode: 404,
+        timestamp: new Date().toISOString(),
```

Comment:
is this relevant?

Reply:
Yes, but I can remove.

Reply:
No need to remove it, i was just curious.